app/models/task.py

```diff
@@ -1,5 +1,46 @@
-from sqlalchemy.orm import Mapped, mapped_column
+from sqlalchemy.orm import Mapped, mapped_column, relationship
 from ..db import db
+from datetime import datetime
+from typing import Optional
+from flask import make_response, abort
+from sqlalchemy import ForeignKey
 
 class Task(db.Model):
     id: Mapped[int] = mapped_column(primary_key=True, autoincrement=True)
+    title: Mapped[str]
```

Comment:
As with `Goal`, consider which of the columns for `Task` should be nullable and which should not be.